addons/hr_expense/models/product_template.py

```diff
@@ -17,6 +17,7 @@ def default_get(self, fields):
 
     can_be_expensed = fields.Boolean(string="Expenses", compute='_compute_can_be_expensed',
         store=True, readonly=False, help="Specify whether the product can be selected in an expense.")
+    purchase_ok = fields.Boolean('Purchase', default=True, compute='_compute_purchase_ok', store=True, readonly=False)
```

Comment:
`can_be_purchased` may be better ?

Reply:
this is the name of the original field in the parent model

Reply:
oooh ok, mb